app/about_us/OurCrew.js

```diff
@@ -0,0 +1,26 @@
+import styles from "./OurCrew.module.css";
+import { ourCrewAbout, ourCrewData } from "./crew.js";
+
+export const OurCrew = () => {
+  return (
+    <section className={styles.container}>
+      <header className={styles.crewHeader}>
+        <p>{ourCrewAbout.description}</p>
+      </header>
+      <div className={styles.crewAbout}>
+        {ourCrewData.map((member, index) => (
+          <div key={index} className={styles.crewItem}>
+            <div className={styles.crewImage}>
+              <img src={member.image} alt={member.name} />
+            </div>
+            <div className={styles.crewInfo}>
+              <h3>{member.title}</h3>
```

Comment:
Logically we should always follow the decreasing order of the Heading tags. `h1` > `h2` > `h3` > ...
But it is okay to modify the size using CSS if needed.

Reply:
Also be careful, since we already have an `<h2>` for this entire section, which is the "Our crew". So these should be a `<h3>` and `<h4>`, following the logic.

Reply:
Make sense!